src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitations/GetInvitationsQueryHandlerTests.cs

```diff
@@ -477,6 +478,22 @@ public async Task Handler_ShouldFilterOnPunchOutInNextWeek()
             }
         }
 
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnPunchOutInNextWeek_ShouldReturnInvitation()
+        {
+            TimeService.SetProvider(new ManualTimeProvider(new DateTime(2019, 10, 25, 11, 0, 0, DateTimeKind.Utc)));
```

Comment:
You already have the ManualTimeProvider from base-class.
Delete this line and use _timeProvider.ElapseWeeks(N); where N is number of weeks. I guess you should have -1